deployment/amazon-ec2/conf/defaults.yml

```diff
@@ -21,7 +21,7 @@ instance_type: m4.xlarge
 image: ami-05cf2265
 volume_type: standard
 key_name: metron-test #Change this
-ec2_host_tag: metron-test
+ec2_host_tag: metron-include-test
```

Comment:
What is this new tag for?  Each EC2 host already gets tagged with 'ec2'.  Do we need another?